### PR TITLE
Blog: ZAPit

### PR DESCRIPTION
**Pull request by psiinon**
<details><summary>Screenshot</summary>
<p>

Screenshot 2023-10-18 at 13-26-07 ZAPit

</p>
</details>

### CODE REVIEW

File: site/content/blog/2023-10-18-zapit/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+./zap.sh -cmd -zapit https://www.example.com
+```
+
+### Feedback
+
+Do you think ZAPit will be useful to you?
+ 
+Would you like it to do anything else?
+
+Let us know via the [ZAP User Group]()
```
Comment (kingthorin) [inline site/content/blog/2023-10-18-zapit/index.md:58]: Link ➕

File: site/content/blog/2023-10-18-zapit/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+##### Credits
+
+The social media background [image](/blog/2023-10-18-zapit/images/blog-zapit.png) is "[A bright and powerful lightning bolt streaks across the sky.](https://wallpapers.com/picture/lightning-bolt-pictures-5rer3bwrq1gj9z8f.html)" by [Wallpapers.com](https://wallpapers.com) and is licensed under [cc by 2.0](https://creativecommons.org/licenses/by/2.0/).
```
Comment (kingthorin) [inline site/content/blog/2023-10-18-zapit/index.md:62]: cc should be full caps I believe.

**Comment by psiinon** [comment]
Done 😁  Will update the User Group link to a specific thread once the blog has been posted and I start one 😉

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**